### ISSUE TITLE
Nouvel utilisateur: mail auto, ajouter le nom d'utilisateur

### ISSUE THREAD

**Issue by AnthonyBrunelli** [DISIC/wikidemarches#1037]
Le nom d'utilisateur est indispensable pour se connecter, il faut donc le rappeler dans le mail automatique lors de la création d'un profil.

**Comment by ClemDee** [comment]
Rappel de l'identifiant dans le mail automatique d'un nouvel utilisateur.
@lucaa, @AnthonyBrunelli : à confirmer si le texte proposé vous convient.

**Comment by ClemDee** [comment]
Le texte rajouté proposé est:
> Vous pourrez ensuite vous connecter en utilisant votre identifiant « identifiant ».

**Comment by ClemDee** [comment]
Réouverture de la tâche en attente de confirmation

**Comment by lucaa** [comment]
@AnthonyBrunelli : de ce que je vois dans les modifications de @ClemDee , l'identifiant était déjà transmis à l'utilisateur dans l'émail de _validation de compte_, même avant les modifications faites pour ce ticket. Tu pensais à quel email / scénario d'inscription dans la description de ce ticket? Y-a-t-il un scénario d'inscription qui ne reçoit pas l'email de validation (ou qui reçoit un email différent qui ne contient pas l'identifiant?)

**Comment by AnthonyBrunelli** [comment]
Je pense que j'ai été un peu approximatif avec le vocabulaire - je parle du mail de création de compte en fait.

**Comment by jrenauxdinum** [comment]
![image](https://user-images.githubusercontent.com/90912308/135278054-7472a0fd-a2ac-431f-a506-ab576922d641.png)

**Comment by jrenauxdinum** [comment]
![image](https://user-images.githubusercontent.com/90912308/135278154-5ecdf754-e99d-4d8f-94b2-d19de1215d5e.png)

**Comment by jrenauxdinum** [comment]
Hello, 
Lorsqu'on a fait des tests avec Anthony début septembre, voici les mails que nous recevions (pour valider la demande d'ouverture de compte (1e capture d'écran) puis pour nous informer de l'acceptation de notre compte (2e capture d'écran).
Il n'y avait pas de rappel de l'identifiant. C'est une question que l'on me pose souvent au téléphone

**Comment by AnthonyBrunelli** [comment]
PI, on est bien ici dans le cas d'une adresse avec un nom de domaine listé (ie. automatique, pas de validation de nous).

**Comment by AnthonyBrunelli** [comment]
PI, test effectué, @ClemDee a fait le test dans l'autre cas, pas de nom Utilisateur non plus.

**Comment by AnthonyBrunelli** [comment]
Mettre le nom d'utilisateur dans les DEUX mails : celui de confirmation ET celui de création de compte.

**Comment by AnthonyBrunelli** [comment]
...et donc sur l'écran de validation de compte.

**Comment by AnthonyBrunelli** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/90912308/135278054-7472a0fd-a2ac-431f-a506-ab576922d641.png)

Correction:

Bonjour XXX,

Cette adresse e-mail a été utilisée pour créer un compte sur le site observatoire.numerique.gouv.fr.

Afin de valider votre compte, cliquez sur le lien suivant :
xxxxxxxxxxxxxx

Votre nom d’utilisateur est XXXXX. Il sera demandé à la connexion.

–
L’Observatoire de la qualité des démarches en ligne

**Comment by AnthonyBrunelli** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/90912308/135278154-5ecdf754-e99d-4d8f-94b2-d19de1215d5e.png)

Correction:

[Prénom Nom]

Votre compte sur l’Observatoire de la qualité des démarches en ligne a bien été activé. 

Votre nom d’utilisateur est : XXX
[Cliquez ici pour vous connecter à l’Observatoire](https://observatoire.numerique.gouv.fr/login/XWiki/XWikiLogin?xredirect=%2FMain%2F ).

–
L’Observatoire de la qualité des démarches en ligne

**Comment by AnthonyBrunelli** [comment]
(point d'attention: le lien dirige vers la page de connexion)

**Comment by AnthonyBrunelli** [comment]
Vous avez sinon le texte/copie d'écran de la page de validation de création de compte? @ClemDee

**Comment by ClemDee** [comment]
@AnthonyBrunelli tu parles des textes de validation / création de compte _avant_ la modification prévue dans cette tâche ?

Si c'est le cas voici par exemple ce que j'avais reçu lors de mon inscription (copié / collé arrangé de mes mails) : 

### Mail d'après création
```
Bonjour Clément Desableau,

Cette adresse e-mail a été utilisée pour enregistrer un nouveau compte sur le site observatoire.numerique.com. Merci d'ignorer ce message si vous n'êtes pas à l'origine de la demande.

Afin de valider votre compte, cliquez sur le lient suivant :
http://observatoire.numerique.com/XWiki/AccountValidation?validkey=&xwikiname=cdesableau

--
L’observatoire de la qualité des démarches en ligne
```

### Mail d'après validation
```
 Madame, Monsieur,

Votre compte observatoire.numerique.gouv.fr est à présent actif. Vous pouvez vous authentifier depuis l'adresse ci-dessous.
http://observatoire.numerique.com/Main/

--
L’observatoire de la qualité des démarches en ligne 
```

**Comment by AnthonyBrunelli** [comment]
Oui, c'est sur ces mails là qu'on propose des corrections - et c'est tout ce qu'il y a à faire dans cette tâche.
Comme je le disais plus haut, je pense qu'il y a une incompréhension du fait que j'ai très mal rédigé mon premier commentaire.
J'ai un doute sur le fait qu'on se comprenne bien, dis-moi s'il y a quelque chose de pas clair.

**Comment by ClemDee** [comment]
Non ça va c'est clair, en fait je me demandais simplement pourquoi est-ce vous aviez besoin des anciens textes alors que vous avez déjà écris les textes de remplacement ! :)

**Comment by AnthonyBrunelli** [comment]
Je ne sais pas si c'est testable - si vous avez comme précédemment le contenu du mail, on prend.

**Comment by AnthonyBrunelli** [comment]
Vu avec @ugodtn c'est bon!
